proxy/api/src/http/error.rs

```diff
@@ -100,6 +100,11 @@ pub async fn recover(err: Rejection) -> Result<impl Reply, Infallible> {
                         "GIT_ERROR",
                         format!("Internal Git error: {:?}", git_error),
                     ),
+                    coco::Error::PathNotFound(path) => (
+                        StatusCode::NOT_FOUND,
+                        "NOT_FOUND",
+                        format!("object not found under '{}'", path),
```

Comment:
I think the message could be a bit more user-friendly, since the only object we currently allow the user to navigate to via the UI is a file, it could read:

"File /{} not found on the current branch"

Currently it looks like this.
<img width="421" alt="Screenshot 2020-09-21 at 09 12 16" src="https://user-images.githubusercontent.com/158411/93740336-92adbc80-fbea-11ea-81bf-665ffae9614b.png">

Reply:
Waiting for input see https://github.com/radicle-dev/radicle-upstream/pull/937#issuecomment-695008704

Reply:
I think something like:

File `<file>` was not found on `<branch>`

.. would do it